src/themes/website/styles/gql.scss

```diff
@@ -0,0 +1,363 @@
+@import "variables.scss";
+
+.table-ws-gql {
+    display: grid;
+    max-width: 100%;
+    overflow-x: auto;
+    margin-bottom: 10px;
+
+    &.logs {
+        grid-template-columns: auto auto 1fr;
+        grid-auto-rows: max-content;
+        background: $form-control-bg-color;
+
+        &>.d-contents>.d-contents> :is(div, pre) {
+            padding: 4px;
+            vertical-align: middle;
+        }
+    }
+}
+
+.gql-ws-spinner {
+    margin: 0px;
+    background: $form-control-bg-color;
+}
+
+.gql-container {
+    display: grid;
+
+    @media (max-width: 768px) {
+        grid-template-columns: 100%;
+        grid-template-areas: "explorer""query""response";
+    }
+
+    @media (max-width: 1200px) and (min-width: 769px) {
+        grid-template-columns: 30% 70%;
+        grid-template-areas: "explorer query"". response";
+    }
+
+    @media (min-width: 1201px) {
+        grid-template-columns: 20% 40% 40%;
+        grid-template-areas: "explorer query response";
+    }
+}
+
+.gql-explorer-grid {
+    grid-area: explorer;
+
+    @media (max-width: 1200px) and (min-width: 769px) {
+        grid-column: 1;
+        grid-row: 1 / 3;
+    }
+
+    h4 {
+        margin-top: 0;
+    }
+
+    @media (min-width: 769px) {
+        padding-right: 20px;
+    }
+
+    @media (max-width: 768px) {
+        margin-bottom: 20px;
+    }
+}
+
+.gql-query-grid {
+    grid-area: query;
+
+    @media (min-width: 1201px) {
+        padding-right: 20px;
+    }
+
+    h4 {
+        margin-top: 0;
+    }
+}
+
+.gql-response-grid {
+    grid-area: response;
+
+    h4 {
+        margin-top: 0;
+    }
+
+    @media (max-width: 1200px) {
+        margin-top: 15px;
+    }
+}
+
+.operation-pointer {
+    cursor: pointer;
+}
+
+.graphql-console-width {
+    @media (min-width: $breakpoint-md) {
+        max-width: 1680px;
+    }
+}
+
+.query-editor {
+    @media (max-height: 1080px) {
```

Comment:
`@media (max-height: 1080px) {`  - a good candidate for variables, like `$breakpoint-...`.